tests/test_patches.py

```diff
@@ -801,7 +1583,7 @@ def test_patches():
     print(f"Botocore version: {botocore.__version__}")
 
     success = True
-    for obj, digests in _API_DIGESTS.items():
+    for obj, digests in _API_DIGESTS:
```

Comment:
can you add an assert that we don't encounter the same item twice, also a comment about why this is a tuple and not dict, otherwise someone may try converting it back to a dict later

Reply:
cmt should go at top of declaration

Reply:
The test now uses pytest parametrization. Does that address your concerns?